tests/Persistence/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ add_executable(Test_TxBody Test_TxBody.cpp)
 target_include_directories(Test_TxBody PUBLIC ${CMAKE_SOURCE_DIR}/src)
 target_link_libraries(Test_TxBody PUBLIC Crypto AccountData Utils Persistence Message)
 
+add_executable(Test_Diagnostic Test_Diagnostic.cpp)
+target_include_directories(Test_Diagnostic PUBLIC ${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/tests)
+target_link_libraries(Test_Diagnostic PUBLIC Crypto AccountData Utils Persistence Message TestUtils)
```

Comment:
Test_Diagnostic should be added to line 48, otherwise it will not be executed in travis verification

Reply:
Done